SECURITY.md

```diff
@@ -0,0 +1,3 @@
+Security issues should not be publicly disclosed unless this has been agreed to by the core Mintlayer team.
+
+Security issues should be reported to security@mintlayer.org or one of the security team.
```

Comment:
I'm not a native speaker so feel free to ignore this but is the end of that sentence correct? It sounds weird.